Backtrace.podspec

```diff
@@ -9,7 +9,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "Backtrace"
-  s.version      = "1.7.5"
+  s.version      = "1.7.6"
```

Comment:
can we bump it to 1.8.0 (minimum). If we have breaking changes, can we bump it to 2.0.0?

Reply:
2.0.0 sounds great

Reply:
Because the library loosing support for iOS/mac/tv 10 I think we should go with 2.0.0